CHANGELOG.md

```diff
@@ -1,13 +1,22 @@
 # bedrock-vue-organization-form ChangeLog
 
-## 1.1.0 - xxxxxx
+## 2.0.0 -
 
 ### Added
-- Added new entity types.
+- Add new entity types.
+- Add Eslint and dependencies to the project.
+
+### Removed
+- Deprecate components such as q-layout-header.
+- Deprecate css classes such as gutter-md.
+
+### Changed
+- Upgrade bedrock-quasar to version 3.0.0.
+- Upgrade Font Awesome to version 5.
+- Change karma config Quasar umd to the latest umd.
 
 ## 1.0.0 - 2019-02-27
 
 ### Added
 - Add core files.
-
```

Comment:
Keep this space ... that last comment isn't an "Added" comment. We've got it in many of our changelogs as just a standalone comment at the end.